cms/djangoapps/contentstore/views/library.py

```diff
@@ -72,21 +72,21 @@ def library_handler(request, library_key_string=None):
         log.exception("Attempted to use the content library API when the libraries feature is disabled.")
         raise Http404  # Should never happen because we test the feature in urls.py also
 
-    if not get_library_creator_status(request.user):
-        if not request.user.is_staff:
-            return HttpResponseForbidden()
+    if request.method == 'POST':
+        if not get_library_creator_status(request.user):
+            if not request.user.is_staff:
```

Comment:
Nit: this check isn't needed here since it's checked within `get_library_creator_status`.